nautobot_device_lifecycle_mgmt/api/serializers.py

```diff
@@ -7,7 +7,7 @@
     NestedPlatformSerializer,
 )
 from nautobot.extras.api.serializers import (
-    CustomFieldModelSerializer,
+    CustomFieldModelSerializerMixin,
```

Comment:
```suggestion
    CustomFieldModelSerializer,
```

Reply:
Should this import not be dropped as your other suggestion would already import it from customfields?

Reply:
Good eye @bajansen. Just sprinted through and did effectively a find and replaced.